charts/rbac/values.yaml

```diff
@@ -1,4 +1,19 @@
 rbac:
+  client:
+    # -- client name (single word, no spaces, dashes, columns), will be used in RBAC group naming.
+    name: "producebox"
```

Comment:
Do we really need to set default value for client name as `producebox`? Let's leave this field commented, cause I'm pretty sure that somebody could suddenly use chart with defaults and create roles for `producebox`. It will always force user to define client `name` if we comment it.

Reply:
fixed